examples/single-service/deploy.yaml

```diff
@@ -1,11 +1,19 @@
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"
+---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   creationTimestamp: null
   labels:
     app: test
   name: test
+  namespace: single-service
```

Comment:
this namespace is a different one than the one created

Reply:
good catch!